scripts/paper_evaluations.yml

```diff
@@ -73,6 +73,23 @@ training_checkpoint_sweep:
         - key: passingBehavior
           value: avoid-pass-alive-territory
 
+# Experiment: evaluate adversary vs. KataGo checkpoints.
+katago_ckpt_sweep:
+  num_games_per_matchup: 50
+  # Adversary config
+  adversary_algorithm: AMCTS
+  adversary_visits: 600
+  adversary_path: "/nas/ucb/k8/go-attack/victimplay\
+    /ttseng-avoid-pass-alive-coldstart-39-20221025-175949/models\
+    /t0-s545065216-d136760487/model.bin.gz"  # cyclic-adv-s545m
```

Comment:
we use `/shared/` in a few places in this `yml` and `/nas/ucb/k8/go-attack/` here — it'd be nicer to consistently use one and do the appropriate conversions  inside `generate_paper_evaluations.py`. I'd say using `/nas/ucb/k8/go-attack/` consistently makes more sense since it's where the models are actually located relative to the machine the developer is using

Reply:
Changed to consistently using `/nas/ucb/k8/go-attack/`. Also checked the generated configs against the old version of the code to make sure my fixes don't actually change the generated configs.